src/internal/events/events.go

```diff
@@ -261,6 +263,12 @@ func Inc(cat m, keys ...string) {
 	metrics.IncrCounter(cats, 1)
 }
 
+// IncN increments the given category by N.
+func IncN(n int, cat m, keys ...string) {
```

Comment:
nit: Can we change `m` to something like `metricCategory` for better readability?